python/cugraph/proto/structure/bicliques.py

```diff
@@ -0,0 +1,297 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Import needed libraries
+import cudf
+import numpy as np
+from collections import OrderedDict
+
+
+def find_bicliques(
+        df, k,
+        offset=0,
+        max_iter=-1,
+        support=1.0,
+        min_features=1,
+        min_machines=10):
+    """
+    Find the top k maximal bicliques
+
+    Parameters
+    ----------
+    df :  cudf:DataFrame
+        A dataframe containing the bipartite graph edge list
+        Columns must be called 'src', 'dst', and 'flag'
+
+    k  :  int
+        The max number of bicliques to return
+        -1 mean all
+
+    offset : int
+
+    Returns
+    -------
+    B : cudf.DataFrame
+        A dataframe containing the list of machine and features.  This is not
+        the full edge list to save space.  Since it is a biclique, it is ease
+        to recreate the edges
+
+        B['id']    - a cluster ID (this is a one up number - up to k)
+        B['vert']  - the vertex ID
+        B['type']  - 0 == machine, 1 == feature
+
```

Comment:
```
B['id'] : cudf.Series
    A cluster ID (this is a one up number - up to k).
```
To be consistent with the cugraph documentation. Same for B['vert'], B['type'], S['id'], ... , S['bad_ration'].